src/components/display.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return <h3>{result}</h3>;
+};
+
+export default Display;
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
```

Comment:
## Add default props in this component
- [ ] This display component should have the incoming prop which is the `result` as a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))
```suggestion
Display.defaultProps = {
  value: "0"
};

Display.propTypes = {
  result: PropTypes.string,
};
```